benchmark.go

```diff
@@ -39,8 +40,36 @@ import (
 var (
 	errNegativeDuration = errors.New("duration cannot be negative")
 	errNegativeMaxReqs  = errors.New("max requests cannot be negative")
+
+	// using a global quantiles array mainly for ease of testing, and not passing the same array
+	// around to multiple functions
+	_quantiles = []string{"0.5000", "0.9000", "0.9500", "0.9900", "0.9990", "0.9995", "1.0000"}
```

Comment:
another note, and doesn't need to be addressed in this PR, with `.` in the field, it makes json parsing (e.g. using `jq` a bit troublesome, but still possible, e.g. we'd `jq `.latencies[".0.9900"]`. Maybe leave a TODO so we make it easier to parse (i.e. output p50, p95, p99 instead)

Reply:
good point. I think we should make the underlying quantile data agnostic to the output format. e.g., just use `[]float64` here, and then the output function can decide how best to format it